src/flow.ts

```diff
@@ -519,6 +529,20 @@ export class Flow {
     localFlags[index] = flags & ~flag;
   }
 
+  setFieldFlag(name: string, flag: FieldFlags): void {
+    let fieldFlags = this.fieldFlags;
+    const flags = fieldFlags.get(name) || 0;
```

Comment:
`Map#get` will result in a runtime error if it doesn't contain `name` when compiling to Wasm because the map's value type is non-nullable but the return value would be nullable (`undefined` in JS). Recommended pattern is to use `assert(Map#get(...))` after `Map#has` returned `true`.